### DISCUSSION TITLE
Espace de discussion sur le site Air travel

### DISCUSSION THREAD

**Discussion by Dackouo** [Dackouo/Exercice#3] [General]
Nous allons discuter ici.

**Comment by Dackouo** [comment]
@Mimi-sang Tu as envoyé tes modifications sur la branche principale, tu devais l'envoyer sur la branche Groupe1. N'oublie pas le git checkout Groupe1 la prochaine fois. J'ai des soucis à effectuer le commit.

**Reply by Mimi-sang** [reply]
D'accord j'ai compris

**Reply by Dackouo** [reply]
D'accord ça marche. Du courage.

**Comment by Dackouo** [comment]
@RahmaWalet Je te reçois.

**Reply by RahmaWalet** [reply]
D'accord merci
